monitoring/uss_qualifier/utils.py

```diff
@@ -0,0 +1,26 @@
+from typing import Optional
+from urllib.parse import urlparse
+
+from monitoring.monitorlib.typing import ImplicitDict
+from monitoring.uss_qualifier.rid.utils import RIDQualifierTestConfiguration
+from monitoring.uss_qualifier.scd.utils import SCDQualifierTestConfiguration
+
+
+class USSQualifierTestConfiguration(ImplicitDict):
+  locale: str
```

Comment:
I would suggest that the sole purpose of this parameter be to locate external files to be used with this qualifier run.